test/handlers_test.go

```diff
@@ -16,9 +16,9 @@ func Test_ValidateDeployRequest_ValidCharacters(t *testing.T) {
 		value    string
 	}{
 		{"lower", "abz"},
-		{"upper", "ABZ"},
-		{"upper and lower mixed", "AbZ"},
```

Comment:
These two should be moved into the invalid characters test.. no need to delete them, they are valid negative test scenarios.